### DISCUSSION TITLE
Release 6.0.1 Sunbird ED Regression Signoff (OCI env)

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [Sunbird-Ed/Community#729] [General Discussion]
Hi All,

This is the sign-off for R6.0.1 for Sunbird-ED as We have completed the new feature verification and Regression testing on the Sunbird-ED OCI environment, Please find the detailed test case execution sheet [Here](https://docs.google.com/spreadsheets/d/1tLk9vTM0oGz33Vck_8msPIlTW8V5T2YV3Bx0AbaNV5Y/edit?pli=1#gid=1232303739&range=A1:R1)

Environment: Sunbird-ED OCI
Testing completion percentage: 100% Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10653)

Scope | Status | Remarks
-- | -- | --
Scope delivered and signed off for the release (Functional + technical) | Done | Delivered Scope : [272 Items](https://docs.google.com/spreadsheets/d/1tLk9vTM0oGz33Vck_8msPIlTW8V5T2YV3Bx0AbaNV5Y/edit?gid=1079242972#gid=1079242972&range=D1)
List of Tech items not tested by QA to be shared | Done | [91 Items](https://docs.google.com/spreadsheets/d/1tLk9vTM0oGz33Vck_8msPIlTW8V5T2YV3Bx0AbaNV5Y/edit?gid=547141331#gid=547141331&range=D1)
List of items descoped | Done | UCI, Lesson Plan functionality's are descoped in this release
All defects to be in closed status (regression and new feature) | Done | All defects are fixed and verified by QA, few report related defects are converted to bugs, which will fix in upcoming release
Call out all defects which have been converted to bugs | Done | [11 Items](https://project-sunbird.atlassian.net/issues/?jql=issuetype+in+%28Bug%2C+Defect%29+AND+status+in+%28Assigned%2C+Closed%2C+Done%2C+%22Failed+Validation%22%2C+Fixed%2C+%22In+Development%22%2C+%22In+Progress%22%2C+%22In+Validation%22%2C+Invalid%2C+Open%2C+%22Pending+Deployment%22%2C+%22Ready+for+Development%22%2C+%22Ready+for+Release%22%2C+Reopen%2C+Reopened%2C+%22Selected+for+Contribution%22%2C+%22To+Do%22%2C+%22Validate%22%29+AND+affectedVersion+in+%285.3.0%2C+5.4.0%2C+5.5.0%2C+5.7.0.0%2C+6.0.0%2C+release-5.1.0%29+AND+created+%3E%3D+2023-10-01+AND+created+%3C%3D+2024-08-30+AND+labels+%3D+OCI_Issue+AND+status+%3D+Open)
Unit Test report for all New Features to be shared | Partially done | Respective TO's have signed off for their Building blocks, but haven't called out the exact % to QA
For an end user facing story, does it have developer documents in place | Done | [Release Notes](https://ed.sunbird.org/use/release/release-notes)
Deployment tracker to be shared and signed off from DC | Done | [Installation](https://ed.sunbird.org/use/getting-started/install) / [upgrade document](https://ed.sunbird.org/use/release/updating-sunbird-releases/5.2.0-to-6.0.0)

Also please find the Regression + New feature sign-off sheets from all the Building Blocks [here](https://docs.google.com/spreadsheets/d/1wG_i_Ix7Udrur2-dtwFq3r2lltNftMApyLHieAGv_94/edit?userstoinvite=naveen.m@qualitrix.com&sharingaction=manageaccess&role=writer#gid=0&range=E1).

![image](https://github.com/Sunbird-Ed/Community/assets/58033030/3fa0098a-3358-4751-b5c4-a711df3fcde8)

CC :
Sunbird ED @vinukumar-vs @mohitga 
Sunbird Knowlg @pallakartheekreddy @mohitga 
Sunbird Inquiry @mohitga @krgauraw
Sunbird CoKreate @vinukumar-vs @mohitga
Sunbird Lern @rhwarrier @AmiableAnil
Sunbird Observ @rhwarrier @anandp504

DC's: @SanthoshVasabhaktula @vrayulu

Sunbird QA: @ashokreddy1208
@gandham-santhosh @santhosh-tg @sanketnagdive @kameshbhr
@madhucr @alok-os @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @kameshbhr @mmanimegalai @manju4201 @rajeevsathish  @anisha147 @swayangjit @Hari-stackroute @Abhign [@kiranharidas187](https://github.com/kiranharidas187) [@baljeetsingh](https://github.com/baljeetsingh) @[siddhishinde0723](https://github.com/siddhishinde0723) [@parthlawate](https://ekstep.slack.com/team/U0HB3HJ73) [@manojLondhe](https://github.com/manojLondhe) [@swayangjit](https://github.com/swayangjit) [@vinod303](https://github.com/vinod303) @PoornimaRajesh @maheshkumargangula @Priyakashi @Naveenkumar739 @raju1307 @Preethi-Sl

**Comment by ddevadat** [comment]
@ashokreddy1208 i am looking to deploy sunbird Ed for an adopter in Oracle cloud ( OCI). Which release version document should i follow. This discussion says regression on 6.0.1 signed off for OCI. Can i use the documentation for release 7.0.0 for deployment in OCI?

**Reply by ravismula** [reply]
@ddevadat 

The latest version (7.5) of Sunbird Ed has transitioned from the traditional setup to Kubernetes. This change has significantly simplified and accelerated the setup and upgrading processes.

However, the official Sunbird release of the easy-installer is currently limited to Azure. We are actively seeking community contributions to extend support to other cloud providers.

We are committed to guiding you through the process and providing support throughout.

Here’s the link to the official repository:
[https://github.com/project-sunbird/sunbird-ed-installer/](https://github.com/project-sunbird/sunbird-ed-installer/)

cc: @rhwarrier @mohitga

**Comment by mohitga** [comment]
Hi Ravi,

Can you help Deepak, with the latest version to deploy on Oracle cloud.
